src/components/Comment/Comment.tsx

```diff
@@ -207,116 +287,133 @@ export const Comment = ({
         </span>
       )}
       <div id={`comment-${comment.id}`} className={commentWrapper}>
-        <div className={avatarMargin}>
-          <Avatar
-            imageUrl={comment.userProfile.avatar}
-            displayName={comment.userProfile.displayName}
-            size={isReply ? "small" : "medium"}
-          />
-        </div>
 
-        <div className={commentDetails}>
-          <header className={headerStyles}>
-            <Column>
-              <Row>
-                <div className={commentProfileName(pillar)}>
-                  <a
-                    href={joinUrl([
-                      "https://profile.theguardian.com/user",
-                      comment.userProfile.userId
-                    ])}
-                    className={linkStyles}
-                  >
-                    {comment.userProfile.displayName}
-                  </a>
-                </div>
-                <div className={timestampWrapperStyles}>
-                  <Timestamp
-                    isoDateTime={comment.isoDateTime}
-                    linkTo={joinUrl([
-                      // Remove the discussion-api path from the baseUrl
-                      baseUrl
-                        .split("/")
-                        .filter(path => path !== "discussion-api")
-                        .join("/"),
-                      "comment-permalink",
-                      comment.id.toString()
-                    ])}
-                  />
-                </div>
-              </Row>
-              <Row>
-                {!!comment.userProfile.badge.filter(
-                  obj => obj["name"] === "Staff"
-                ).length ? (
-                  <div className={iconWrapper}>
-                    <GuardianStaff />
+        {/* Default view */}
+        <div className={css`
+          ${until.mobileLandscape} {
+            display: none;
+          }
+        `}>
+          <div className={flexRowStyles}>
+            <div className={cx(marginRight, flexGrow)}>
+              <Avatar
+                imageUrl={comment.userProfile.avatar}
+                displayName={comment.userProfile.displayName}
+                size={isReply ? "small" : "medium"}
+              />
+            </div>
+            <div className={flexColStyles}>
+              <header className={cx(flexRowStyles, headerStyles, fullWidthStyles)}>
+                <div className={flexColStyles}>
+                  <div className={cx(flexRowStyles, alignItemsCenter)}>
+                    <div className={marginRight}>
+                      <ProfilName
```

Comment:
I don't think there's any benefit in this abstraction. I'd be in favour of less DRY and more repetition to make the code more readable and maintainable.

Reply:
good point, I hoped it would make it easier to read, but it doesnt add much